include/wx/dataobj.h

```diff
@@ -427,6 +427,10 @@ class WXDLLIMPEXP_CORE wxTextDataObject : public wxDataObjectSimple
 #endif // different wxTextDataObject implementations
 
 private:
+#if defined(__WXQT__)
+	void DoSetDataFrom(const class QMimeData &mimeData, const wxDataFormat &format) wxOVERRIDE;
```

Comment:
Here and elsewhere in this PR the indentation is wrong (2 spaces instead of 4).

Reply:
Ah, I have tabs instead of spaces by default, I will change :)